EKrut-Client/src/ekrut/client/managers/ClientInventoryManager.java

```diff
@@ -63,6 +88,12 @@ public void updateItemThreshold(Item item, String ekrutLocation, int threshold)
 			throw new Exception(resultCode); // TBD CHANGE TO SPESIFIC EXCEPTION?
 	}
 	
+	/**
+	 * Send Client's request to the server, returns the server's response.
+	 * 
+	 * @param request the InventoryItemRequest instance representing the request.
+	 * @return InventoryItemResponse the server's response for the given request.
```

Comment:
No need to specify the return type in this tag, just a description of the value being returned.